tests/next_tests/integration_tests/feature_tests/ffront_tests/test_decorator.py

```diff
@@ -0,0 +1,35 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
This is something for the type checker and not a regression test in my opinion.
```suggestion
# TODO(dropd): Remove as soon as `gt4py.next.ffront.decorator` is type checked.
```

Reply:
That would be much preferred, indeed.